easy_phi/tests/scpi2widgets_test.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+import tempfile
+
+from tornado.test.util import unittest
+from tornado.options import options
+
+from easy_phi import scpi2widgets
+
+
+class BasicWidgetsTest(unittest.TestCase):
+    reset_widget = "reset widget"
+    sysname_widget = "system name widget"
+    container = "#blablah"
+    conf = None  # link to NamedTempFile to prevent deletion
+
+    def setUp(self):
+        widgets_conf = tempfile.NamedTemporaryFile()
+        widgets_conf.write("""[DEFAULT]
+
+[RESET]
+scpi = *RST
+widget = {reset_widget}
+
+[System name]
+scpi = SYSTem:NAME?
+widget = {sysname_widget}\n""".format(reset_widget=self.reset_widget,
+                                      sysname_widget=self.sysname_widget))
+        widgets_conf.flush()
+        self.conf = widgets_conf
```

Comment:
this line costs about 2 hours of work, yet does not make any sense on first look. Without it, widget_conf removed by garbage collector after setUp finished